### ISSUE TITLE
Editing Bidirectional Relationships

### ISSUE THREAD

**Issue by DerBeutlin** [girzel/ebdb#94]
Very minor issue regarding https://github.com/girzel/ebdb/issues/86

A short example
I have two contacts Alice and Bob,
I add a relationship of Bob as the boyfriend of Alice, everything works fine due to your work in the other issue, and Alice becomes the girlfriend of Bob.
Now they marry and I want to update their relationship.
Now if I edit Alice's relationship to Bob (from Alice's entry) I cannot update the relationship while if I do it from Bob's entry I can update it.

Finally there is probably potential for auto-guessing the corresponding role if you keep track of the gender of the contact, but most probably that is not worth the hassle

**Comment by girzel** [comment]
Huh, I thought for sure I had provided for  "reverse editing" relationships, I will check this out tomorrow, thanks for the report.

**Comment by girzel** [comment]
What I'm seeing is that you can edit the relationship field from either record, but the prompts ("this label", "the other label") don't change depending on which record you're on. ie, the label prompts are "backwards" if you're editing from the record that doesn't actually own the field instance.

Is that consistent with what you're seeing?

**Comment by DerBeutlin** [comment]
Yes I tested it again, and it seems to be that way. Sorry I actually did it because I did the role association wrong with brother and sister and wanted to change it and probably I then did not look at the labels and did it wrong again and therefore it did not change. 
Sorry for the wrong issue description.

**Comment by girzel** [comment]
Okay, no problem -- this is a real issue (it has annoyed me in other circumstances), but one that's going to take a little bit of thought. Let me sit on this for a bit.
